github-actions/action.yml

```
@@ -23,6 +23,18 @@ inputs:
     required: false
     default: |-
       third-party/**/*
+  branch:
+    description: |-
+      The branch name used by the bot to be pushed at the origin.
+    default: generated-doc-github-actions
+  commit_title:
+    description: |-
+      The title of the commit with the generated doc. The PR will also use this
+      title.
+    default: 'doc: auto-generated GH Actions README.md'
```

```suggestion
    default: 'doc: auto-generated GitHub Actions README.md'
```
For clarity, since it's quite possible that's the first interaction some people have with this.

---

Sounds a good idea.